src/react-native/rollbar.js

```diff
@@ -23,7 +23,12 @@ function Rollbar(options, client) {
   delete this.options.maxItems;
   this.options.environment = this.options.environment || 'unspecified';
   var api = new API(this.options, transport, urllib);
-  this.client = client || new Client(this.options, api, logger, 'react-native');
+
+  // remove tracer object from options so we don't try to parse it
+  var tracer = options.tracer || null;
+  delete options.tracer;
+  this.client = client || new Client(this.options, api, logger, 'react-native', tracer);
```

Comment:
Logic looks similar to what's in src/browser/rollbar.js. Maybe worth refactoring into a shared function to set this.client

Reply:
Each implementation (node, react-native, browser) don't inherit from some shared class so their instantiation of the client is unique to their implementation so I don't think a shared function would make sense since the client has to live as an attribute to the individual implementation.